MeNext.Spotify/SpotifyMusicService.cs

```diff
@@ -57,92 +63,166 @@ public IResultList<IPlaylist> UserLibraryPlaylists
         {
             return webApi.metadata.GetSongs(uids);
         }
 
+        /// <summary>
+        /// Gets the artists.
+        /// </summary>
+        /// <returns>The artists.</returns>
+        /// <param name="uids">list of artist ids.</param>
         public IList<IArtist> GetArtists(IList<string> uids)
         {
             return webApi.metadata.GetArtists(uids);
         }
 
+        /// <summary>
+        /// Gets the albums.
+        /// </summary>
+        /// <returns>The albums.</returns>
+        /// <param name="uids">list of album ids> .</param>
         public IList<IAlbum> GetAlbums(IList<string> uids)
         {
             return webApi.metadata.GetAlbums(uids);
         }
 
+        /// <summary>
+        /// Gets the playlists.
+        /// </summary>
+        /// <returns>The playlists.</returns>
+        /// <param name="uids">list of playlist ids.</param>
         public IList<IPlaylist> GetPlaylists(IList<string> uids)
         {
             return webApi.metadata.GetPlaylists(uids);
         }
 
+        /// <summary>
+        /// Searchs for the album.
+        /// </summary>
+        /// <returns>The album.</returns>
+        /// <param name="query">Query for album.</param>
         public IResultList<IAlbum> SearchAlbum(string query)
         {
             return webApi.SearchAlbum(query);
         }
 
+        /// <summary>
+        /// Searchs for the artist.
+        /// </summary>
+        /// <returns>The artist.</returns>
+        /// <param name="query">Query for artist.</param>
         public IResultList<IArtist> SearchArtist(string query)
         {
             return webApi.SearchArtist(query);
         }
 
+        /// <summary>
+        /// Searchs for the playlists.
+        /// </summary>
+        /// <returns>The playlist.</returns>
+        /// <param name="query">Query for playlist.</param>
         public IResultList<IPlaylist> SearchPlaylists(string query)
         {
             return webApi.SearchPlaylists(query);
         }
 
+        /// <summary>
+        /// Searchs the song.
+        /// </summary>
+        /// <returns>The song.</returns>
+        /// <param name="query">Query for song.</param>
         public IResultList<ISong> SearchSong(string query)
         {
             return webApi.SearchSong(query);
         }
 
+        /// <summary>
+        /// Adds the status listener.
+        /// </summary>
+        /// <param name="listener">Listener to add.</param>
         public void AddStatusListener(IMusicServiceListener listener)
         {
             listeners.Add(listener);
         }
 
+        /// <summary>
+        /// Removes the status listener.
+        /// </summary>
+        /// <param name="listener">Listener to remove.</param>
         public void RemoveStatusListener(IMusicServiceListener listener)
         {
             listeners.Remove(listener);
         }
 
+        /// <summary>
+        /// Songs the ends.
```

Comment:
Seriously? This is just the default text. You obviously didn't read it. I'm not wasting effort on more like this. Check your work.